CSS/QUIZPAGE/style.css

```diff
@@ -1,11 +1,9 @@
 main {
+  height:100%;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  /* padding-left: 10px;
-  padding-right: 10px;
```

Comment:
You can remove the comments

Reply:
Thank you 
The comment is already removed. Red Lines are the removals